### ISSUE TITLE
Download video from local manifest.mpd

### ISSUE THREAD

**Issue by davispuh** [ytdl-org/youtube-dl#25326]
## Checklist

- [x] I'm reporting a feature request
- [x] I've verified that I'm running youtube-dl version **2020.05.08**
- [x] I've searched the bugtracker for similar feature requests including closed ones

## Description

It seems that currently you can't download video from local manifest.mpd file.

For example I have locally this `manifest.mpd` file with content:
```xml
<?xml version="1.0"?>
<MPD xmlns="urn:mpeg:dash:schema:mpd:2011" minBufferTime="PT1.500S" type="static" mediaPresentationDuration="PT0H0M3.515S" maxSegmentDuration="PT0H0M2.248S" profiles="urn:mpeg:dash:profile:isoff-on-demand:2011,http://dashif.org/guidelines/dash264">
    <Period duration="PT0H0M3.515S">
        <AdaptationSet segmentAlignment="true" maxWidth="368" maxHeight="640" maxFrameRate="19200/681" par="276:480" lang="und" subsegmentAlignment="true" subsegmentStartsWithSAP="1">
            <Representation id="174356317284970v" mimeType="video/mp4" codecs="avc1.4D401E" width="276" height="480" frameRate="19200/608" sar="1:1" startWithSAP="1" bandwidth="251438">
                <BaseURL>https://example.org/video_low.mp4</BaseURL>
                <SegmentBase indexRangeExact="true" indexRange="953-1008">
                    <Initialization range="0-952" />
                </SegmentBase>
            </Representation>
            <Representation id="277572850281398v" mimeType="video/mp4" codecs="avc1.4D0029" width="368" height="640" frameRate="19200/608" sar="1:1" startWithSAP="1" bandwidth="679305">
                <BaseURL>https://example.org/video_good.mp4</BaseURL>
                <SegmentBase indexRangeExact="true" indexRange="940-995">
                    <Initialization range="0-939" />
                </SegmentBase>
            </Representation>
        </AdaptationSet>
        <AdaptationSet segmentAlignment="true" lang="und" subsegmentAlignment="true" subsegmentStartsWithSAP="1">
            <Representation id="549754235974131a" mimeType="audio/mp4" codecs="mp4a.40.5" audioSamplingRate="44100" startWithSAP="1" bandwidth="51816">
                <AudioChannelConfiguration schemeIdUri="urn:mpeg:dash:23003:3:audio_channel_configuration:2011" value="1" />
                <BaseURL>https://example.org/audio.mp4</BaseURL>
                <SegmentBase indexRangeExact="true" indexRange="912-967">
                    <Initialization range="0-911" />
                </SegmentBase>
            </Representation>
        </AdaptationSet>
    </Period>
</MPD>
```

It contains video for low and high quality and also audio. So I want to download highest quality and merge it with audio.
Basically `youtube-dl ./manifest.mpd` (or something like `youtube-dl file://./manifest.mpd`)

Current workaround seems to be just host that file with `python -m http.server 8000` and then use `youtube-dl http://127.0.0.1:8000/manifest.mpd` which while seems to work isn't that nice.

**Comment by davispuh** [comment]
If you mark it as duplicate then you should link that request because I can't find it.
